pkg/systemd/services/services.jsx

```diff
@@ -749,46 +584,76 @@
         }
     }
 
-    render() {
-        const { unit_by_path } = this.state;
-        const path = this.props.path;
+    // compute filtered and sorted list of [Id, unit]
+    computeSelectedUnits() {
+        if (!this.props.activeTab)
+            return [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230303-065136-93ef1f6a-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)